xds/internal/xdsclient/xdslbregistry/xdslbregistry_test.go

```diff
@@ -203,6 +203,27 @@ func (s) TestConvertToServiceConfigSuccess(t *testing.T) {
 			wantConfig: `[{"round_robin": {}}]`,
 			pfDisabled: true,
 		},
+		{
+			name: "pick_first_enabled_pf_rr_use_pick_first",
+			policy: &v3clusterpb.LoadBalancingPolicy{
+				Policies: []*v3clusterpb.LoadBalancingPolicy_Policy{
+					{
+						TypedExtensionConfig: &v3corepb.TypedExtensionConfig{
+							TypedConfig: testutils.MarshalAny(&v3pickfirstpb.PickFirst{
+								ShuffleAddressList: true,
+							}),
+						},
+					},
+					{
+						TypedExtensionConfig: &v3corepb.TypedExtensionConfig{
+							TypedConfig: testutils.MarshalAny(&v3roundrobinpb.RoundRobin{}),
+						},
+					},
+				},
+			},
+			wantConfig: `[{"pick_first": { "shuffleAddressList": true }}]`,
+			pfDisabled: false,
```

Comment:
Nit: we can omit this line as the default (zero value) for a boolean is `false`, unless you feel that explicitly including this line here improves the readability of the test.

Reply:
Done. Thanks!